habitat-lab/setup.py

```diff
@@ -36,6 +36,13 @@
     "Bug Tracker": "https://github.com/facebookresearch/habitat-lab/issues",
 }
 
+EXTRA_DEPS = {
+    # Home robot dependency for velocity controllers
+    "continuous_control": [
```

Comment:
Why call it continuous_control ? 
All rearrangement tasks are using continuous control and none of them use home-robot. 
Suggestions : 
 - continuous_navigation
 - home-robot
 - continuous_control_for_navigation

Reply:
We might add more controllers from home-robot in the future so using home_robot is accurate in my opinion